test/WireMock.Net.Tests/WireMockServerTests.cs

```diff
@@ -196,6 +198,92 @@ public async Task WireMockServer_When_HttpClientWithWebProxyCallsHttp_Should_Wor
     }
 #endif
 
+#if NET6_0_OR_GREATER
+    [IgnoreOnContinuousIntegrationFact]
+    public async Task WireMockServer_WithUrl0000_Should_Listen_On_All_IPs_IPv4()
+    {
+        // Arrange
+        var IPv4 = new List<string>();
+
+        foreach (var netInterface in NetworkInterface.GetAllNetworkInterfaces())
+        {
+            if (netInterface.OperationalStatus == OperationalStatus.Up)
+            {
+                var ipProps = netInterface.GetIPProperties();
+                foreach (var addr in ipProps.UnicastAddresses)
+                {
+                    if (addr.Address.AddressFamily == System.Net.Sockets.AddressFamily.InterNetwork)
+                    {
+                        IPv4.Add(addr.Address.ToString());
+                    }
+                }
+            }
+        }
```

Comment:
refactor to:
```
private static string[] GetIPAddressesByFamily(AddressFamily addressFamily)
    {
        return NetworkInterface.GetAllNetworkInterfaces()
            .Where(ni => ni.OperationalStatus == OperationalStatus.Up)
            .SelectMany(ni => ni.GetIPProperties().UnicastAddresses)
            .Where(addr => addr.Address.AddressFamily == addressFamily)
            .Select(addr => addr.Address.ToString())
            .ToArray();  // Change from ToList() to ToArray()
    }
    ```

Reply:
I currently added it near the top in the test file WireMockServerTests.cs, but would it be better to move it to an own utils file?
Maybe use this one for it: TestUtils.cs?

Reply:
- use `#if NET6_0_OR_GREATER` for both unit tests
- also include that new helper method inside the if-def